salt/core/locale.sls

```diff
@@ -0,0 +1,20 @@
+## Locale
+add_system_locale_en_gb_utf8:
+  # This line may be in the file but commented out.
+  # However we use file.append here instead of file.replace because we can't guarantee each Linux Distro build will comment out the lines in the same way.
+  # Could use replace with "- append_if_not_found: True"
```

Comment:
Should we just do that?

Reply:
I've updated this file to use file.replace with append_if_not_found. 
This is slightly better as it removes the risk of it getting set twice.